Purpose: make sure the ISO gets unmounted correctly when booted via findiso/toram

--- a/scripts/live
+++ b/scripts/live
@@ -1684,7 +1684,7 @@
 	then
 		losetup -d /dev/loop0
 
-		if is_mountpoint /live/findiso
+		if is_mountpoint /root/live/findiso
 		then
 			umount /root/live/findiso
 			rmdir --ignore-fail-on-non-empty /root/live/findiso \
